lib/index.ts

```diff
@@ -384,6 +375,8 @@ export default class MaplibreGeocoder {
    * const GeoApi = {
    *   forwardGeocode: (config) => { return { features: [] } },
    *   reverseGeocode: (config) => { return { features: [] } }
+   *   getSuggestions: (config) => { return { suggestions: {text: string, placeId?: string}[] }}
```

Comment:
The example here is an "actual" example, there should be something like `place: "my-place",...` I think.

Reply:
I removed this as I think including these optional functions does not help the goal of showing how to attach the geocoder to a container element.

Reply:
I would recommend adding a "simple" example and "complicated" example.